etna/transforms/scalers.py

```diff
@@ -23,6 +24,7 @@ def __init__(
         inplace: bool = True,
         with_mean: bool = True,
         with_std: bool = True,
+        mode: str = TransformMode.per_segment,
```

Comment:
type TransformMode

Reply:
Here we pass the sting, not the TransformMode

Reply:
Done

Reply:
```
>>> from etna.transforms.sklearn import TransformMode
>>> TransformMode.per_segment
<TransformMode.per_segment: 'per-segment'>
>>> type(TransformMode.per_segment)
<enum 'TransformMode'>
```
?

Reply:
There should be "per-segment" instead of TransformMode.per_segment)